omod/src/main/resources/liquibase.xml

```diff
@@ -195,5 +195,31 @@
 							 tableName="metadatamapping_metadata_source"
 							 columnNames="name" />
 	</changeSet>
-
+	
+	<changeSet id="metadatamapping-2016-01-06-0800" author="jasonvena">
+		<preConditions onFail="MARK_RAN">
+			<columnExists tableName="metadatamapping_metadata_term_mapping"
+						  columnName="metadata_class" />
+		</preConditions>
+		<comment>
+			Make MetadataTerm.metadataClass optional
+		</comment>
+		<dropNotNullConstraint tableName="metadatamapping_metadata_term_mapping"
+							   columnName="metadata_class"
+							   columnDataType="varchar(1024)" />
+	</changeSet>
+	
+	<changeSet id="metadatamapping-2016-01-06-0801" author="jasonvena">
+		<preConditions onFail="MARK_RAN">
+			<columnExists tableName="metadatamapping_metadata_term_mapping"
+						  columnName="metadata_uuid" />
+		</preConditions>
+		<comment>
+			Make MetadataTerm.metadataUuid optional
```

Comment:
Same as above, we refer to the table name (rather than the class name) in the hibernate mapping, please see [MetadataTerm.hbm.xml](https://github.com/openmrs/openmrs-module-metadatamapping/blob/df90370adde14251bdef0538c8f3ddb8b0fd2544/api/src/main/resources/MetadataTerm.hbm.xml#L19).